pkg/bindings/amqp/message.go

```diff
@@ -0,0 +1,91 @@
+package amqp
+
+import (
+	"fmt"
+	"reflect"
+	"strings"
+
+	"github.com/cloudevents/sdk-go/pkg/binding/format"
+	"github.com/cloudevents/sdk-go/pkg/binding/spec"
+	ce "github.com/cloudevents/sdk-go/pkg/cloudevents"
+	"github.com/cloudevents/sdk-go/pkg/cloudevents/types"
+	"pack.ag/amqp"
+)
+
+const attrPrefix = "cloudEvents:" // Attribute key prefix
```

Comment:
Are these the AMQP Application Property Values? Perhaps different constant name ?